twentytwentyone-blocks/assets/css/blocks.css

```diff
@@ -243,4 +243,16 @@ h1.wp-block-site-title a:not(:hover):not(:focus):not(:active) {
 
 .wp-block-social-links.is-style-twentytwentyone-social-icons-color .wp-social-link {
 	background: none;
+}
+
+/*--------------------------------------------------------------
+# Table Block
+--------------------------------------------------------------*/
+
+.wp-block-table {
+	min-width: 240px;
```

Comment:
I wonder if we still need this. @kjellr what do you think?

Reply:
Yeah, probably fine to remove. Gutenberg doesn't set a min-width itself, but I think that's ok. 👍